.github/workflows/continuous_integration.yml

```diff
@@ -15,14 +15,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - name: Set up Python 3.8
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v2
         with:
-          python-version: 3.8
+          python-version: "3.8"
       - name: Install dependencies
         run: |
-          pip install -U pip
+          pip install -U pip setuptools wheel
```

Comment:
Seems like the base branch is out of date, mind merging master into both branches so that we get a clean diff?